[2024-12-05T18:20:10.653+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T18:20:10.664+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL.trigger_dag_extract_tables manual__2024-12-05T17:55:02.823737+00:00 [queued]>
[2024-12-05T18:20:10.670+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL.trigger_dag_extract_tables manual__2024-12-05T17:55:02.823737+00:00 [queued]>
[2024-12-05T18:20:10.671+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-05T18:20:10.680+0000] {taskinstance.py:2889} INFO - Executing <Task(TriggerDagRunOperator): trigger_dag_extract_tables> on 2024-12-05 17:55:02.823737+00:00
[2024-12-05T18:20:10.687+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=794) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T18:20:10.688+0000] {standard_task_runner.py:72} INFO - Started process 796 to run task
[2024-12-05T18:20:10.688+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'ETL', 'trigger_dag_extract_tables', 'manual__2024-12-05T17:55:02.823737+00:00', '--job-id', '370', '--raw', '--subdir', 'DAGS_FOLDER/etl.py', '--cfg-path', '/tmp/tmpvj3e1see']
[2024-12-05T18:20:10.690+0000] {standard_task_runner.py:105} INFO - Job 370: Subtask trigger_dag_extract_tables
[2024-12-05T18:20:10.721+0000] {task_command.py:467} INFO - Running <TaskInstance: ETL.trigger_dag_extract_tables manual__2024-12-05T17:55:02.823737+00:00 [running]> on host 886abab866d2
[2024-12-05T18:20:10.904+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='ETL' AIRFLOW_CTX_TASK_ID='trigger_dag_extract_tables' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T17:55:02.823737+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T17:55:02.823737+00:00'
[2024-12-05T18:20:10.905+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T18:20:11.728+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T18:20:11.729+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=ETL, task_id=trigger_dag_extract_tables, run_id=manual__2024-12-05T17:55:02.823737+00:00, execution_date=20241205T175502, start_date=20241205T182010, end_date=20241205T182011
[2024-12-05T18:20:11.784+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T18:20:11.794+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-05T18:20:11.797+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
